ui/pages/home/home.component.js

```diff
@@ -550,6 +554,94 @@ export default class Home extends PureComponent {
     );
   };
 
+  renderOpenSeaProviderPopover() {
+    const { t } = this.context;
+    const { setTransactionSecurityCheckEnabled } = this.props;
+
+    return (
+      <Popover
+        title={
+          <Typography
+            variant={TYPOGRAPHY.H4}
+            color={COLORS.TEXT_ALTERNATIVE}
+            fontWeight={FONT_WEIGHT.BOLD}
+          >
+            {t('staySafeWithOpenSea')}
+          </Typography>
+        }
+        footer={
+          <>
+            <Button
+              type="primary"
+              onClick={() => {
+                setTransactionSecurityCheckEnabled();
+                setOpenSeaTransactionSecurityProviderPopoverHasBeenShown();
+              }}
+              className="home__enable-security-provider-button"
+            >
+              {t('enableOpenSeaSecurityProvider')}
+            </Button>
+            <Box marginTop={2}>
+              <Typography variant={TYPOGRAPHY.H6}>
+                <Button
+                  type="link"
+                  onClick={() =>
+                    setOpenSeaTransactionSecurityProviderPopoverHasBeenShown()
+                  }
+                >
+                  {t('notNow')}
+                </Button>
+              </Typography>
+            </Box>
+          </>
+        }
+        footerClassName="smart-transactions-popover__footer"
+        className="smart-transactions-popover"
+        onClose={() =>
+          setOpenSeaTransactionSecurityProviderPopoverHasBeenShown()
+        }
+      >
+        <Box
+          paddingRight={6}
+          paddingLeft={6}
+          paddingTop={0}
+          paddingBottom={0}
+          display={DISPLAY.FLEX}
+          className="smart-transactions-popover__content"
+        >
+          <Box
+            marginTop={1}
+            marginBottom={1}
+            display={DISPLAY.FLEX}
+            flexDirection={FLEX_DIRECTION.COLUMN}
+          >
+            <img
+              src="./images/open-sea-security-provider.svg"
+              alt={t('openSeaAltText')}
+            />
+          </Box>
+          <Typography variant={TYPOGRAPHY.H6}>
+            {t('getWarningsFromOpenSea')}
+          </Typography>
+          <Typography
+            variant={TYPOGRAPHY.H6}
+            marginTop={4}
+            fontWeight={FONT_WEIGHT.BOLD}
+          >
+            {t('openSeaDescription')}
+          </Typography>
+          <Typography
+            variant={TYPOGRAPHY.H7}
+            color={COLORS.TEXT_ALTERNATIVE}
+            marginTop={4}
+          >
+            {t('alwaysBeSureTo')}
+          </Typography>
+        </Box>
+      </Popover>
+    );
+  }
+
```

Comment:
Home component is hugs, lets extract this into separate component.

Reply:
Fixed here: b693772adac938c037cc398851c79bc16a7c0610